src/db.rs

```diff
@@ -0,0 +1,56 @@
+use crate::dcl_hex;
+use crate::pb::dcl;
+use crate::utils::sanitize_sql_string;
+use substreams::prelude::BigInt;
+use substreams_database_change::pb::database::{table_change, DatabaseChanges};
+
+pub fn transform_collection_database_changes(
+    changes: &mut DatabaseChanges,
+    collections: dcl::Collections,
```

Comment:
is it required to use the namespace here?

Reply:
I could use just `Collections` but I'd need to modify the top level import to 
```
use crate::{dcl_hex, pb::dcl::Collections};
```
so at the end of the day, I think it's quite similar to use any of these 2 alternatives